packages/rocket-auth-aws-infrastructure/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@boostercloud/rocket-auth-aws-infrastructure",
+  "version": "0.9.4",
```

Comment:
Note that we're on `0.10.0` now

Reply:
Fixed [here](https://github.com/boostercloud/booster/pull/556/commits/dfc0c4014bcdfd27086220b92a91a30fe6acd1af)